mq/plugins/guardduty_mapping.yml

```diff
@@ -1048,111 +1048,95 @@
     gdeventlastseents: details.finding.eventLastSeen
+    direction: details.finding.action.networkConnectionAction.connectionDirection
+    apiname: details.finding.apicalls.name
+    accesskeyid: details.resource.accessKeyDetails.accessKeyId
+    principalid: details.resource.accessKeyDetails.principalId
   
   Recon:IAMUser/NetworkPermissions:
     findingid: details.id
     arn: details.arn
     awsaccountid: details.accountId
+    awsregion: details.region
     resourcetype: details.resource.resourceType
-    accesskeyid: details.resource.accessKeyDetails.accessKeyId
-    principalid: details.resource.accessKeyDetails.principalId
-    usertype: details.resource.accessKeyDetails.userType
-    username: details.resource.accessKeyDetails.userName
     detectorid: details.finding.detectorId
-    apiname: details.finding.apicalls.api
-    sourceipaddress: details.finding.action.awsApiCallAction.remoteIpDetails.ipAddressV4
     gdeventcreatedts: details.createdAt
     gdeventupdatedts: details.updatedAt
     gdeventfirstseents: details.finding.eventFirstSeen
     gdeventlastseents: details.finding.eventLastSeen
+    direction: details.finding.action.networkConnectionAction.connectionDirection
+    apiname: details.finding.apicalls.name
+    accesskeyid: details.resource.accessKeyDetails.accessKeyId
+    principalid: details.resource.accessKeyDetails.principalId
   
   Recon:IAMUser/ResourcePermissions:
     findingid: details.id
     arn: details.arn
     awsaccountid: details.accountId
     awsregion: details.region
     resourcetype: details.resource.resourceType
-    accesskeyid: details.resource.accessKeyDetails.accessKeyId
-    principalid: details.resource.accessKeyDetails.principalId
-    usertype: details.resource.accessKeyDetails.userType
-    username: details.resource.accessKeyDetails.userName
     detectorid: details.finding.detectorId
-    evidence: details.finding.evidence
-    apiname: details.finding.apicalls.api
-    sourceipaddress: details.finding.action.awsApiCallAction.remoteIpDetails.ipAddressV4
     gdeventcreatedts: details.createdAt
     gdeventupdatedts: details.updatedAt
     gdeventfirstseents: details.finding.eventFirstSeen
     gdeventlastseents: details.finding.eventLastSeen
+    direction: details.finding.action.networkConnectionAction.connectionDirection
+    apiname: details.finding.apicalls.name
+    accesskeyid: details.resource.accessKeyDetails.accessKeyId
+    principalid: details.resource.accessKeyDetails.principalId
   
   Recon:IAMUser/UserPermissions:
     findingid: details.id
     arn: details.arn
     awsaccountid: details.accountId
     awsregion: details.region
     resourcetype: details.resource.resourceType
-    accesskeyid: details.resource.accessKeyDetails.accessKeyId
-    principalid: details.resource.accessKeyDetails.principalId
-    usertype: details.resource.accessKeyDetails.userType
-    username: details.resource.accessKeyDetails.userName
```

Comment:
Don't we need userName and userType in a UserPermissions event?

Reply:
@mpurzynski Do these specific types of events not have userName and userType mappings?